lib/generators/rad_common/install/install_generator.rb

```diff
@@ -48,6 +48,9 @@ def create_initializer_file
         copy_file '../../../../../spec/dummy/config/initializers/simple_form_components.rb',
                   'config/initializers/simple_form_components.rb'
 
+        copy_file '../../../../../spec/dummy/config/initializers/wicked_pdf.rb',
```

Comment:
seems like we won't need this file in the projects, when I remove it from better_way, the PDF's still work, I think it can use the initializer in rad_common

Reply:
@garyfoster which initializer in rad_common?

Reply:
see the below comment, I think it needs to be moved

Reply:
its removed